spec/features/repair_request_spec.rb

```diff
@@ -298,15 +298,17 @@ def stub_property_temp_annex
       expect(page).to have_css(".hackney-property-warning-label-turquoise")
       click_on 'Raise a repair on this dwelling'
 
-      expect(page).to have_content "CC"
+      page.should have_css('.hackney-cautionary-contact-table')
       expect(page).to have_link("Launch Keyfax", href: "https://www.keyfax.com")
+      expect(find_by_id('hackney_repair_request_description')).to have_content 'CC'
```

Comment:
`expect(page).to have_field('Description', with: 'CC')`

it also happens on lines 309, 310 and 359

Reply:
Ohhh I see this is actually the text on the work order page, and not the form field.